examples/todo-pylist.html

```diff
@@ -54,7 +50,8 @@ <h1>To Do List</h1>
 
                       # add a new task to the list and tell it to use the `content` key to show in the UI
                       # and to use the key `done` to sync the task status with a checkbox element in the UI
-                      myList.add(task)
+                      myList = Element("myList")
+                      myList.element.pyElementInstance.add(task)
```

Comment:
This is so ugly. Definitely something we want to improve through the plugins API. 

Just leaving this as note to self, not in the scope of this PR.